src/lib/y2storage/proposal/space_distribution.rb

```diff
@@ -36,6 +36,12 @@ class SpaceDistribution
 
       include Yast::Logger
 
+      # Space that will be consumed by the EBR if we create a new extended
+      # partition. Value based on libstorage-ng behavior
+      # See https://en.wikipedia.org/wiki/Extended_boot_record
+      OVERHEAD_OF_NEW_EXTENDED = DiskSize.MiB(2)
```

Comment:
Shouldn't this number come out of libstorage, like based on the current alignment policy?